### PR TITLE
[Nano] Correct typos in ReadtheDocs

### PR DESCRIPTION
**Pull request by Oscilloscope98**
## Description

Correct typos, correct grammatical errors, and refactor some sentences in readthedocs for Nano section.

Test link: https://yuwentestdocs.readthedocs.io/en/correct-docs-typo-nano/doc/Nano/Overview/nano.html

### CODE REVIEW

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ trainer.test(ort_model, dataloader)
 trainer.predict(ort_model, dataloader)
 ```
 ### ONNXRuntime Acceleration
-Before you start with onnxruntime accelerator, you are required to install some onnx packages as follows to set up your environment with ONNXRuntime acceleration.
+Before you start with ONNXruntime accelerator, you are required to install some ONNX packages as follows to set up your environment with ONNXRuntime acceleration.
```
Comment (shane-huang) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md:40]: should it be ONNXRuntime?

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md [hunk 2]
```diff
@@ -179,11 +179,11 @@ A set of arguments that helps to tune the results for both INC and POT quantizat
...
 **Accuracy Control with INC**
-There are a few arguments that require only by INC, and you should not specify or modify any of them if you use `accelerator='openvino'`.
-- `tuning_strategy`(optional): it specifies the algorithm to search the tuning space. In most cases, you don't need to change it.
-- `timeout`: Timeout of your tuning. Defaults 0 means endless time for tuning.
+There are a few arguments required only by INC, and you should not specify or modify any of them if you use `accelerator='openvino'`.
+- `tuning_strategy` (optional): it specifies the algorithm to search the tuning space. In most cases, you don't need to change it.
+- `timeout`: Timeout of your tuning. Defaults `0` means endless time for tuning.
 
-Here is an example to use INC with accuracy control as below. It will search for a model within 1% accuracy drop with 10 trials.
+Here is an example to use INC with accuracy control as below. It will search for a model within 1% accuracy drop within 10 trials.
```
Comment (shane-huang) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md:186]: `with` seems more accurate in this context.

File: docs/readthedocs/source/doc/Nano/QuickStart/tensorflow_inference.md [hunk 3]
```diff
@@ -67,12 +68,12 @@ A set of arguments that helps to tune the results for both INC and POT quantizat
...
 **Accuracy Control with INC**
-There are a few arguments that require only by INC.
-- `tuning_strategy`(optional): it specifies the algorithm to search the tuning space. In most cases, you don't need to change it.
+There are a few arguments required only by INC.
+- `tuning_strategy` (optional): it specifies the algorithm to search the tuning space. In most cases, you don't need to change it.
 - `timeout`: Timeout of your tuning. Defaults 0 means endless time for tuning.
 - `inputs`:      A list of input names. Default: None, automatically get names from the graph.
 - `outputs`:     A list of output names. Default: None, automatically get names from the graph.
-Here is an example to use INC with accuracy control as below. It will search for a model within 1% accuracy drop with 10 trials.
+Here is an example to use INC with accuracy control as below. It will search for a model within 1% accuracy drop within 10 trials.
```
Comment (shane-huang) [inline docs/readthedocs/source/doc/Nano/QuickStart/tensorflow_inference.md:76]: with

Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/QuickStart/tensorflow_inference.md]: Have changed back

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md (hunk 1 above)
Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md]: Have fixed.

File: docs/readthedocs/source/doc/Nano/Overview/nano.md [hunk 4]
```diff
@@ -45,11 +45,11 @@ In a pure pip environment, you need to run `source bigdl-nano-init` every time y
 
 #### **3.1 PyTorch**
 
-BigDL-Nano supports both PyTorch and PyTorch Lightning models and most optimizations requires only changing a few "import" lines in your code and adding a few flags.
+BigDL-Nano supports both PyTorch and PyTorch Lightning models and most optimizations require only changing a few "import" lines in your code and adding a few flags.
 
 BigDL-Nano uses a extended version of PyTorch Lightning trainer for integrating our optimizations.
 
-For example, if you are using a LightingModule, you can use the following code to enable intel-extension-for-pytorch and multi-instance training.
+For example, if you are using a LightningModule, you can use the following codes to enable intel-extension-for-pytorch and multi-instance training.
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Nano/Overview/nano.md:52]: I think code is not countable when used in the context of programming. Maybe "the following lines code" or the "the following code snippet"

File: docs/readthedocs/source/doc/Nano/Overview/nano.md [hunk 5]
```diff
@@ -76,11 +76,11 @@ For more details on the BigDL-Nano's PyTorch usage, please refer to the [PyTorch
 
 ### **3.2 TensorFlow**
 
-BigDL-Nano supports `tensorflow.keras` API and most optimizations requires only changing a few "import" lines in your code and adding a few flags.
+BigDL-Nano supports `tensorflow.keras` API and most optimizations require only changing a few "import" lines in your codes and adding a few flags.
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Nano/Overview/nano.md:79]: same as above

File: docs/readthedocs/source/doc/Nano/Overview/nano.md [hunk 6]
```diff
@@ -76,11 +76,11 @@ For more details on the BigDL-Nano's PyTorch usage, please refer to the [PyTorch
...
 
 BigDL-Nano uses a extended version of `tf.keras.Model` or `tf.keras.Sequential` for integrating our optimizations.
 
-For example, you can conduct a multi-instance training using the following code:
+For example, you can conduct a multi-instance training using the following codes:
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Nano/Overview/nano.md:83]: same as above

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md [hunk 7]
```diff
@@ -154,7 +154,7 @@ trainer.validate(ov_q_model, dataloader)
 trainer.test(ov_q_model, dataloader)
 trainer.predict(ov_q_model, dataloader)
 ```
-Same as ONNXRuntime, it equals to converting the model from Pytorch to OpenVINO firstly and then doing quantization on the converted OpenVINO model:
+Same as quantization using INC with ONNXRuntime, it equals to converting the model from PyTorch to OpenVINO firstly and then doing quantization on the converted OpenVINO model:
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md:157]: How about "Same as using onnxruntime accelerator" ?

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_onnxruntime.md [hunk 8]
```diff
@@ -86,4 +86,4 @@ y_hat = ort_model(x)
 y_hat.argmax(dim=1)
 ```
 - Note
-    `ort_model` is not trainable any more, so you can't use like trainer.fit(ort_model, dataloader)
\ No newline at end of file
+    `ort_model` is not trainable any more, so you can't use a code like `trainer.fit(ort_model, dataloader)`
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_onnxruntime.md:89]: how about "cannot use it in `fit` such as `trainer.fit(ort_model, dataloader)`"?

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md [hunk 9]
```diff
@@ -1,15 +1,15 @@
...
+For quantization, BigDL-Nano provides only post-training quantization in `trainer.quantize()` for users to infer with models of 8-bit precision. Quantization-aware training is not available for now. Model conversion to 16-bit like BF16, and FP16 will be coming soon.
 
 Before you go ahead with these APIs, you have to make sure BigDL-Nano is correctly installed for PyTorch. If not, please follow [this](../Overview/nano.md) to set up your environment.
 
 ##  Runtime Acceleration
-All available runtime accelerations are integrated in `Trainer.trace(accelerator='onnxruntime'/'openvino')` with different accelerator value. Before you know about BigDL-Nano and any optimizations on inference, taking mobilenetv3 as an example, you may have one script for training and inference like this:
+All available runtime accelerations are integrated in `Trainer.trace(accelerator='onnxruntime'/'openvino')` with different accelerator values. Before diving into BigDL-Nano and any optimizations on inference, you may have one script for training and inference like this, taking mobilenetv3 as an example model:
```
Comment (yangw1234) [inline docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md:12]: How about
“All available runtime accelerations are integrated in `Trainer.trace(accelerator='onnxruntime'/'openvino')` with different accelerator values. Let's take mobilenetv3 as an example model and here is a short script that you might have before applying any BigDL-Nano's optimizations.”

File: docs/readthedocs/source/doc/Nano/Overview/nano.md (hunk 4 above)
Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/Overview/nano.md]: Have fixed it :D

File: docs/readthedocs/source/doc/Nano/Overview/nano.md (hunk 5 above)
Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/Overview/nano.md]: Have fixed it

File: docs/readthedocs/source/doc/Nano/Overview/nano.md (hunk 6 above)
Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/Overview/nano.md]: Have fixed it

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md (hunk 7 above)
Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_inference.md]: Agree :D Have changed it

File: docs/readthedocs/source/doc/Nano/QuickStart/pytorch_onnxruntime.md (hunk 8 above)
Comment (Oscilloscope98) [reply docs/readthedocs/source/doc/Nano/QuickStart/pytorch_onnxruntime.md]: Have changed it

**Review by yangw1234 — APPROVED** [review]
LGTM

**Comment by shane-huang** [comment]
LGTM

**Review by shane-huang — APPROVED** [review]
